Assets/Scripts/Managers/HighlightManager.cs

```diff
@@ -31,6 +37,18 @@ public static void OnEntityClicked(EntityClass clicked)
         if (CombatManager.Instance.GameState != GameState.SELECTION) return;
         bool isOutlined = false;
 
+        Debug.Log(clicked.GetType());
+
+        if (clicked is PlayerClass)
+        {
+            if ((PlayerClass)clicked != currentPlayerDisplayDeck)
+            {
+                currentPlayerDisplayDeck?.UnRenderHand();
+            }
+            currentPlayerDisplayDeck = (PlayerClass) clicked;
+            ((PlayerClass)clicked).PleaseRenderMyHand();
+        }
```

Comment:
Might want to use pattern matching here:
`if (clicked is PlayerClass player)
{
    if (player != currentPlayerDisplayDeck)
    {
        currentPlayerDisplayDeck?.UnRenderHand();
    }
    currentPlayerDisplayDeck = player;
    player.PleaseRenderMyHand();
}`